LabBuilder/LabBuilder.psm1

```diff
@@ -2896,6 +3012,7 @@ function Initialize-LabVMImage {
     # Apply an unattended setup file
     Write-Verbose -Message $($LocalizedData.ApplyingVMBootDiskFileMessage `
         -f $VM.Name,'Unattend','Unattend.xml')
+		New-Item -Path "$MountPoint\Windows\Panther" -ItemType Directory -Force -InformationAction SilentlyContinue
```

Comment:
I actually fixed this line in the dev branch. I'm thinking this might be where your merge conflict is. Are you able to remove your change?